packages/lib/gpcircuits/package.json

```diff
@@ -29,10 +29,13 @@
     "prepublishOnly": "yarn clean && yarn build",
     "test": "ts-mocha --type-check --config ../../../.mocharc.js --exit 'test/**/*.spec.ts'",
     "clean": "rm -rf dist node_modules *.tsbuildinfo build circuits/test ptau && yarn clean-test-artifacts",
-    "gen-circuit-parameters": "npx tsx scripts/gen-circuit-parameters.ts",
-    "gen-test-artifacts": "npx tsx scripts/gen-test-artifacts.ts",
+    "gen-circuit-parameters": "tsx scripts/gen-circuit-parameters.ts",
+    "gen-test-circuit-parameters": "tsx scripts/gen-circuit-parameters.ts test",
+    "gen-artifacts": "tsx scripts/gen-artifacts.ts",
+    "gen-test-artifacts": "tsx scripts/gen-artifacts.ts test",
+    "clean-prod-artifacts": "rm -rf artifacts/prod circuits/main",
```

Comment:
The prod variants imply some new directories which need to appear in .gitignore and turbo.json.